packages/routers/src/CommonActions.tsx

```diff
@@ -70,9 +56,9 @@ export function navigate(...args: any): Action {
   } else {
     const payload = args[0] || {};
 
-    if (!payload.hasOwnProperty('key') && !payload.hasOwnProperty('name')) {
+    if (!payload.hasOwnProperty('name')) {
```

Comment:
We could maybe provide a helpful error for people upgrading from v6 to v7 in here:

```jsx
    if (!payload.hasOwnProperty('name')) {
      let message = 'You need to specify a name ...';

      if (payload.hasOwnProperty('key')) {
        message = 'Helpful message that key was removed in v7. ' + message;
      }

      throw new Error(message);
    }
```

Reply:
I didn't bother to add an error message since we don't really throw for other breaking changes so having errors for only a few isn't of much help. Hopefully, the migration guide + TypeScript will help catch these breaking changes.